packages/react-native/update-android.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-ANDROID_REPO_DIR=../../../bugsnag-android
+ANDROID_REPO_DIR=../../../android
```

Comment:
did you mean to commit this? It's not an ideal setup but most people (myself included) have it checked out in a directory called `bugsnag-android`

Reply:
Yeah this was accidental, sorry. I have all our repos without the `bugsnag` prefix so autocomplete is more helpful